### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by showbit01** [AI4Bharat/NPTEL2020-Indian-English-Speech-Dataset#19]
Hi,
Due to large dataset, I have downloaded the dataset in chunks (3 of 5 gb approx) from academic torrent ,but the text ground truth is not there.Would you please help me in this.

Thanks

**Comment by hayk314** [comment]
I am having the same issue with train dataset stored here. The test and validation parts have text and metadata but the train does not. Is this an expected behavior for the train part? @GokulNC @Prem-kumar27 @snakers4

**Comment by showbit01** [comment]
@hayk314 no,text tokens should be needed as ground truth labels with audio signal as input

**Comment by hayk314** [comment]
Thanks for your reply @showbit01 . Are you saying that text metadata must indeed be present in the train part of the dataset stored here? Just to clarify, my question is that I see the text is there for **test** and **validation**, but not for the **train**. I just want to be sure that there is indeed no text meta available for the train part here or if there is perhaps someone can hint on its whereabouts.

**Comment by HemanthSai7** [comment]
Hey, did you happen to find the manifest files for train set?
@hayk314 @showbit01 @GokulNC
